src/index.js

```diff
@@ -25,9 +25,11 @@ class Node extends libp2p {
           secio
         ]
       },
-      discovery: [
-        webRTCStar.discovery
-      ]
+      discovery: []
+    }
+
+    if (options.webrtc) {
```

Comment:
It needs to be webRTCStar, as there might be other webRTCFlavors.

Is there a PR for js-ipfs to take this into account now?